subprojects/dependency-management/src/testFixtures/groovy/org/gradle/integtests/fixtures/publish/MavenPublishedModule.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.publish
+
+import org.gradle.integtests.fixtures.GradleMetadataResolveRunner
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import org.gradle.test.fixtures.maven.DelegatingMavenModule
+import org.gradle.test.fixtures.server.http.MavenHttpModule
+
+class MavenPublishedModule extends DelegatingMavenModule<MavenHttpModule> {
```

Comment:
:x: This name isn't great.

The reason I used `PublishedModule` for my other fixture is that it is used when the test actually uses Gradle to publish the module, and we use the fixture to verify details of the module. This one is more of a 'TestModule', since it is generated by and consumed by tests. `MavenConsumableModule`? `MavenModuleSpec`?

Reply:
I have removed it, it was leftover from the previous implementation without the repository test fixture.